hedera-node/hedera-app/src/main/java/com/hedera/node/app/ServicesMain.java

```diff
@@ -225,6 +237,9 @@ public static void main(final String... args) throws Exception {
         final var recycleBin =
                 RecycleBin.create(metrics, configuration, getStaticThreadManager(), time, fileSystemManager, selfId);
 
+        // FUTURE: use the new feature flag addressbook.useRosterLifecycle value when it is available
+        final boolean addressbookUseRosterLifecycle = false;
```

Comment:
This feature flag is being added in https://github.com/hashgraph/hedera-services/pull/16445

Reply:
Now that https://github.com/hashgraph/hedera-services/pull/16445 is merged I will update this to use the feature flag from the configuration.

Reply:
The feature flag behavior should be implemented by https://github.com/hashgraph/hedera-services/issues/16357, so I will not be adding it here.